local_planner/src/nodes/star_planner.cpp

```diff
@@ -1,23 +1,23 @@
 #include "star_planner.h"
 
-StarPlanner::StarPlanner() {}
+namespace avoidance {
+
+StarPlanner::StarPlanner() : tree_age_(0) {}
 
 StarPlanner::~StarPlanner() {}
 
 // set parameters changed by dynamic rconfigure
 void StarPlanner::dynamicReconfigureSetStarParams(
-    avoidance::LocalPlannerNodeConfig& config, uint32_t level) {
+    const avoidance::LocalPlannerNodeConfig& config, uint32_t level) {
   childs_per_node_ = config.childs_per_node_;
   n_expanded_nodes_ = config.n_expanded_nodes_;
   tree_node_distance_ = config.tree_node_distance_;
   tree_discount_factor_ = config.tree_discount_factor_;
 }
 
-void StarPlanner::setParams(double min_cloud_size,
-                            double min_dist_backoff,
+void StarPlanner::setParams(double min_cloud_size, double min_dist_backoff,
```

Comment:
I guess these doubles could be passed by reference as well

Reply:
I would not pass doubles by reference I thought it only makes sense really for bigger entities, no?

Reply:
Yeah, pass by reference is just extra complications for small types. No performance differences (in fact, possibly worse), and it makes it harder to read.